### DISCUSSION TITLE
VPN Ipsec route based between a cisco router and  Strongswan

### DISCUSSION THREAD

**Discussion by azyane** [strongswan/strongswan#1018] [Q&A]
Hello everyone,
I'm setting up a VPN Ipsec route based between a cisco router and a Strongswan machine on Amazon linux 2.
The link is established, but I can't connect to the target network 208.224.250.0/24.
Here is my VPN configuration : 
conn tns-preludd
        authby=secret
        auto=start
        type=transport
        reauth=yes
        rekey=yes
        leftprotoport=47
        left=10.100.10.210
        leftid=50.50.50.21
        leftsourceip=10.100.10.210
        leftsubnet=10.100.10.0/24
        right=60.60.60.23
        rightsubnet=208.224.250.0/24
        rightid=60.60.60.23
        rightprotoport=47
        ike=aes256-sha256-modp2048!
        ikelifetime=28800s
        esp=aes256-sha2_256!
        lifetime=1h
        dpddelay=10
        dpdtimeout=60
        keyexchange=ikev2
after I executed the following commands : 
#Add the interface
ip tunnel add prelu_gre local 10.100.10.210 remote 60.60.60.23 mode gre
#Activate it
ip link set prelu_gre up
#Add an IP address
ip route add 208.224.250.0/24 dev prelu_gre
#Set the MTU, to account for GRE/ESP protocol overhead
ip link set dev prelu_gre mtu 1440
Now when i do a traceroute from my Strongswan machine to 208.224.250.2 I have the impression that it seeks to pass by Internet and not to go up to the link Ipsec.
Can you please Help me ?

**Comment by tobiasbrunner** [comment]
You seem to have confused a net-to-net policy-based connection (as indicated by the values in `left|rightsubnet`) with a GRE-based connection (as indicated by the `left|rightprotoport` values and the commands you executed), which would actually require a host-to-host IPsec tunnel between the GRE endpoints. You should decide on the approach you want to use (also depends on what the other end requires because GRE can't be used unilaterally) and then adapt your config accordingly.
